dev-client/src/screens/ProjectListScreen/ProjectListScreen.tsx

```diff
@@ -71,41 +70,30 @@ export const ProjectListScreen = () => {
   );
 
   return (
-    <ScreenScaffold
-      AppBar={
-        <AppBar
-          LeftButton={null}
-          RightButton={<AppBarIconButton name="help" />}
-        />
-      }>
+    <ScreenScaffold AppBar={<AppBar LeftButton={null} />}>
       <VStack
         bg="grey.200"
         p={5}
         flexGrow={1}
         flexShrink={0}
         flexBasis="70%"
         space="10px">
-        <Box alignItems="flex-start" pb={3}>
-          <AddButton
-            text={t('projects.create_button')}
-            buttonProps={{onPress}}
-          />
-        </Box>
-
         {isLoadingData ? (
           <Spinner size="lg" />
         ) : (
           activeProjects.length === 0 && (
-            <>
-              <Text variant="body1-strong">{t('projects.none.header')}</Text>
+            <Box mb={4}>
```

Comment:
Suggestion: Use spacing values like "sm", "md", "lg" from theme.ts, or a string for pixels like `mb="16px"`, rather than these nativebase unknown units, which in my experience have sometimes not corresponded to the same number of pixels